add puma control app

edebill opened the pull request:
App is set to listen on port 9293, which is not piped out of docker so it will need to be accessed via docker exec.

docker exec <container id> bundle exec pumactl --control-url tcp://127.0.0.1:9293 --control-token shoptalk stats

edebill committed:
add puma control app

App is set to listen on port 9293, which is not piped out of docker so
it will need to be accessed via docker exec.

docker exec <container id> bundle exec pumactl --control-url
tcp://127.0.0.1:9293 --control-token shoptalk stats

offbyone commented:
I like this, but I'm holding off a bit longer hoping that I can land the CI change into mainline mastodon first.

offbyone committed:
Enable a custom image prefix for Mastodon image builds

By default the prefix is ghcr.io/mastodon/mastodon, but for forks it
would be useful to be able to specify a different prefix, allowing them
to trivially build their own docker images based off the trunk
